infra/modules/azure_app_service_plan_autoscaler/variables.tf

```diff
@@ -14,16 +14,21 @@ variable "autoscale_name" {
   default     = null
 }
 
-variable "target_service" {
-  type = object({
+variable "target_services" {
```

Comment:
I suggest to change the variable to this structure, even though a breaking change is introduced:

```hcl
type = object({
    app_service_name  = optional(list(string))
    function_app_name = optional(list(string))
})
```

Looks smoother to me :) 

cc @gunzip

Reply:
Agree

Reply:
Fixed in [f8de935](https://github.com/pagopa/dx/pull/190/commits/f8de9355a42d7688b83992c3f5de0ea22ea3bc6b)